src/coreclr/inc/readytorun.h

```diff
@@ -80,6 +80,7 @@ enum class ReadyToRunSectionType : uint32_t
     ComponentAssemblies         = 115, // Added in V4.1
     OwnerCompositeExecutable    = 116, // Added in V4.1
     PgoInstrumentationData      = 117, // Added in 5.2
+    ManifestAssemblyMvids       = 118, // Added in V5.3
```

Comment:
Nit: Add `V` before `5.2`.

Reply:
Fixed in 4th commit, thanks for spotting the inconsistency.